### DISCUSSION TITLE
Integrate gopass into my golang cli tool

### DISCUSSION THREAD

**Discussion by airtonix** [gopasspw/gopass#2694] [Q&A]
I have a cli that takes a config that i'm planning to make like the following if `source === 'gopass-totp` 

```json
{
  "sources": [
    {
      "name": "test",
      "config": {
        "outputFormat": "{{Account.Name}}-{{Date.From}}-{{Date.To}}.csv"
      },
      "credentials": {
        "source": "gopass-totp",
        "usernameKey": "me/place/username",
        "passwordKey": "me/place/password",
        "totpKey": "me/place/totp"
      }
    }
  ]
}
```

Any documentation about using gopass as a go module?

**Comment by dominikschulz** [comment]
Yes, we have a public Go API. It's considered unstable and has a big fat warning, but that's mostly due to the fact that we didn't get a lot of feedback on it so far: https://pkg.go.dev/github.com/gopasspw/gopass@v1.15.8/pkg/gopass

So if you're feeling brave go ahead and use and please make sure to report back.
